x-pack/filebeat/module/fortinet/firewall/manifest.yml

```diff
@@ -1,14 +1,14 @@
 module_version: 1.0
 
 var:
-  - name: syslog_host
+  - name: host
```

Comment:
you might still want to call this variables `syslog_host` and `syslog_port` to stay aligned with the rest of the modules. The underlying protocol is still syslog (more or less), right?